cowprotocol/explorer
# Fix percent formatting

# Summary

I notice that the percentage shown in the FILL tab was wrong. It was showing `>99.99%` when it was 100%

I fixed the issue in this PR, plus implemented a bunch of tests:

<img width="572" alt="image" src="https://user-images.githubusercontent.com/2352112/228967697-ec5424b0-263c-490e-a078-647179a33c61.png">

# To Test

Check how in this PR (https://github.com/cowprotocol/explorer/pull/412) the % of 0x76cb4b2ac25690664ccb2967b5322f17e1d966e31b205cf86a2b1956bf2f5e54dbc682d0ec19bd276fd6c2af4e0189b584bb31e164258bb1 is bad, and in this one is good :)
